lib/controller.js

```diff
@@ -7,7 +7,10 @@ var sprintf = require('sprintf-js').sprintf;
 var cp = require('child_process');
 var async = require('async');
 var updates = require('./update-fetch');
+var debug = require('debug')('controller');
+var Menu = require('terminal-menu');
 var controller = {};
+controller.TESSEL_AUTH_KEY = Tessel.TESSEL_AUTH_PATH + '/id_rsa';
```

Comment:
I commented on the other PR, but I think this variable should actually be defined and exported in `provision.js` since that file is responsible for writing the keys. What do you think?